GemeindescanExporter/metadata.txt

```diff
@@ -1,8 +1,8 @@
 [general]
-name=GemeindescanExporter
-description=Gemeindescan Exporter plugin
-about=Plugin to export data and styles to <a href="https://gemeindescan.ch/de/">gemeindescan</a>
-version=0.0.1
+name=SpatialDataPackageExporter
+description=Spatial Datapackage Exporter plugin
+about=Plugin to export data and styles to a Spatial Data Package compatible with <a href="https://bitbucket.org/cividi/gemeindescan-webui">Gemeindescan</a>.
+version=0.0.2
```

Comment:
CD process changes version number automatically when making release, you don't have to change this.

Reply:
:+1: thanks, will leave it at 0.0.1